mnefun/_fetching.py

```diff
@@ -47,9 +53,23 @@ def fetch_raw_files(p, subjects, run_indices):
                           % remote_fnames[:1])
         # make the name "local" to the acq dir, so that the name works
         # remotely during rsync and locally during copyfile
-        remote_dir = [fn[:fn.index(op.basename(fn))]
-                      for fn in remote_fnames][0]
```

Comment:
> Also, what happens if run "_01" was collected on one date and "_02" was collected on another? In the PR code, remote_dirs can't have multiple listings.

See highlighted code -- I think we only ever used a single directory on `master`, too, the difference is that on `master` we (arbitrarily and silently) chose to use the first directory name rather than choosing one in a principled way. So we can figure out how to use multiple directories in another PR if necessary.